src/mantine-dates/src/components/MonthsList/get-month-tabindex/get-month-tabindex.ts

```diff
@@ -0,0 +1,9 @@
+import dayjs from 'dayjs';
+
+export function getMonthTabIndex(month: Date, months: Date[][]) {
+  if (dayjs(month).isSame(months[0][0], 'month')) {
```

Comment:
- Why is there no logic for selected month?
- Same question as with days – what if first month is disabled?

Reply:
Added the same logic for all levels now!